plugins/modules/site_intent.py

```diff
@@ -90,18 +88,27 @@
               name:
                 description: Name of the floor (eg floor-1).
                 type: str
-              parentName:
-                description: Complete Parent name of the floor to be created(eg Global/USA/San Francisco/BGL_18).
+              parent_name:
+                description: Complete parent name of the floor to be created(eg Global/USA/San Francisco/BGL_18).
                 type: str
               rf_model:
-                description: Type of floor. Allowed values are 'Cubes And Walled Offices',
-                  'Drywall Office Only', 'Indoor High Ceiling', 'Outdoor Open Space'.
+                description: Type of floor (allowed values are 'Cubes And Walled Offices', 'Drywall Office Only', 'Indoor High Ceiling',
```

Comment:
The RF (Radio Frequency) model type for the floor, which is essential for simulating and optimizing wireless network coverage. Select from the following allowed values, which describe different environmental signal propagation characteristics.

Then you can explain all the types

Reply:
updated.